examples/kitchen-sink/packages/eslint-config-custom-server/index.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
   parserOptions: {
     ecmaVersion: "latest",
     sourceType: "module",
+    babelOptions: {
+      presets: [require.resolve('next/babel')],
```

Comment:
We shouldn't need this here, this config is for server apps only and doesn't extend from next